test/gaudi_opts/test_global_converter_maps.py

```diff
@@ -38,7 +38,7 @@
 )
 
 podioInput = PodioInput("InputReader")
-podioInput.collections = ["MCParticles"]
+podioInput.collections = ["MCParticles", "EventHeader"]
```

Comment:
Do we still need the `EventHeader` after merging https://github.com/key4hep/k4FWCore/pull/160 ? If no, then I think we should remove it here.

Reply:
we should be able to remove it and that was my plan for today :)